sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/TablePartition.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.sql.connector.catalog;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TablePartition {
```

Comment:
Partitions are already referred to in other parts of the catalog with the Transform class, do we need this as well?

Reply:
Thanks for your CR.
Transform doest not contains actual partition values and partition metadata. In many cases, we need to know the metadata of a partition.
Please correct me if there is something wrong.

Reply:
Thanks for reply, I will doing it.